index.html

```diff
@@ -383,7 +383,7 @@ <h3>Ed25519VerificationKey2020</h3>
               "id": "https://example.com/issuer/123#key-0",
               "type": "Ed25519VerificationKey2020",
               "controller": "https://example.com/issuer/123",
-              "publicKeyMultibase": "zgo4sNiXwJTbeJDWZLXVn9uTnRwgFHFxcgDePvEC9TiTYgRpG7q1p5s7yRAic"
+              "publicKeyMultibase": "zdbDmZLTWuEYYZNHFLKLoRkEX4sZykkSLNQLXvMUyMB1"
```

Comment:
This looks like it's off by 1-2 bytes (should be 45-46 bytes, I'm seeing 44 bytes)... possibly due to varint encoding for the `0xed` value? A `publicKeyMultibase` value for `Ed25519VerificationKey2020` should start with a 'z' and then a `0xed01` byte header (which is the `0xed` multicodec value encoded as a varint).

@dmitrizagidulin it looks like this bug exists in our code as well: https://github.com/digitalbazaar/ed25519-signature-2020/blob/main/test/mock-data.js#L11

... or was there some decision made somewhere that we're not doing multicodec encoding of multibase public keys anymore? /cc @dlongley

Reply:
Yeah, this looks totally wrong: https://github.com/digitalbazaar/ed25519-verification-key-2020/blob/main/lib/Ed25519VerificationKey2020.js#L104

Reply:
It needs to be fixed to include the multikey prefix.

Reply:
Addressing in #11

Reply:
shouldn't `publicKeyMultibase` match a did:key, for example:

`did:key:z6Mkn9p3p3MEKekdhKQ7SYZ91LjeSUgiYkzin4VPt6BeHNj8`

`z6Mkn9p3p3MEKekdhKQ7SYZ91LjeSUgiYkzin4VPt6BeHNj8` is a base58 encoded ed25519 public key.

for existing key types, I would expect them to be resolvable when prefixed with did:key

Reply:
highly possible I have just not looks at this in a while and am confused, but I just spent all weekend parsing public  key bytes from did key fingerprints... and it would be share for that to be somehow not the same as this.

Reply:
Yes `publicKeyMultibase` should match a `did:key` (what was one of the goals of using `publicKeyMultibase` over base58btc) -- I know DB's implementation is broken in this regard and we're fixing it for the 2020 suite.

Reply:
This means that there are a lists of mostly usable examples here: https://github.com/w3c-ccg/did-method-key/tree/master/test-vectors

hidden inside of the didDocument.id property.

but beware that bls12381 as a format that has 2 public keys in it, because its a pairing friendly curve.

Reply:
The `z6Mk` prefix for Ed25519 public key (like in [did:key](https://w3c-ccg.github.io/did-method-key/#ed25519-x25519)) can be seen in #11